solidity/contracts/OngoingAirdrops.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import './utils/Governable.sol';
+import '../interfaces/IOngoingAirdrops.sol';
+import '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+import '@openzeppelin/contracts/utils/cryptography/MerkleProof.sol';
+
+abstract contract OngoingAirdrops is Governable, IOngoingAirdrops {
+  using SafeERC20 for IERC20;
+
+  /// @inheritdoc IOngoingAirdrops
+  mapping(bytes32 => bytes32) public roots;
+  /// @inheritdoc IOngoingAirdrops
+  mapping(bytes32 => uint32) public deadline;
```

Comment:
Since we are storing it alone, I don't see any benefits in making it `uint32` instead of `uint256`. It might even be more expensive gas-wise to do so

Reply:
They don't